engines/wintermute/ad/ad_actor_3dx.cpp

```diff
@@ -166,13 +166,13 @@ bool AdActor3DX::update() {
 			float walkVel = _directWalkVelocity == 0.0f ? _velocity : _directWalkVelocity;
 			Math::Vector3d newPos = _posVector;
 			if (_directWalkMode == DIRECT_WALK_FW) {
-				newPos.x() += -sinf(_angle.getRadians()) * walkVel * _scale3D * (float)_gameRef->_deltaTime / 1000.f;

```

Comment:
I would suggest making some short comments around a lot of these changes, to document the LH -> RH conversion. In case someone at some later point tries to debug some of this, and checks against original WME, and then get confused.

Reply:
Done